### DISCUSSION TITLE
My domain expired and is under grace period. HOW DO I RENEW IT?? HOW DO I PAY??

### DISCUSSION THREAD

**Discussion by kumarabhirup** [vercel/community#5283] [Help]
### Summary

My card was not charged, please charge my card and help me renew my domains that were set on auto-renewal, I haven't been able to get my domains since 2 weeks now 🙏
My Vercel username: @kumareth

cc @amyegan 🙌

### Steps to Reproduce

- Have a domain on Vercel
- Payment method (card) expires
- Domain renewal fails
- Domain expires
- Can't renew, no UI to do that, seems like it has to be done manually?
- Contact support
- Support doesn't respond 😅

**Comment by amyegan** [comment]
Thanks for waiting, @kumarabhirup! It looks like someone already responded to your support ticket and renewed your domain, so I'm going to close this discussion now.
